authentication_authorization/README.md

```diff
@@ -51,7 +51,7 @@ urlpatterns = patterns('',
 
 Then we need a template for the login page, so create a directory `blog/templates/registration` and a file inside named `login.html`:
 
-```
+```django
```

Comment:
highlight.js does have django support... I also gave it a quick try in the tutorial with gitbook (I can't get extensions running as it's gitbook 1.5 still!) and it seemed to work :smile: